sympy/plotting/plot.py

```diff
@@ -237,7 +237,15 @@ def append(self, arg):
         Plot object containing:
         [0]: cartesian line: x**2 for x over (-10.0, 10.0)
         [1]: cartesian line: x for x over (-10.0, 10.0)
-
+        
+        .. plot::
+
+           import matplotlib.pyplot as plt
+           import matplotlib.image as mpimg
+           import numpy as np
+           img = mpimg.imread('/sympy/doc/src/modules/plotOutputs/append.png')
+           imgplot = plt.imshow(img)
+        
```

Comment:
The purpose of the plot directive is so that the images are automatically generated. You do not need to display a pregenerated image. The code in the plot directive will be run when the docs are built, the image will be generated, and inserted into the documentation. Look at the source files of the beam module examples. There are no image files explicitly included.